virt/driver.go

```diff
@@ -18,8 +18,10 @@ import (
 
 	domain "github.com/hashicorp/nomad-driver-virt/internal/shared"
 	"github.com/hashicorp/nomad-driver-virt/libvirt"
+	"github.com/hashicorp/nomad-driver-virt/virt/idset"
 
 	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/nomad/client/lib/numalib/hw"
```

Comment:
since we import stuff out of `nomad/` anyway, why copy over the `idset.go` file(s)? could just import that package too?

Reply:
I thought about it, but wasn't sure because I wanted to keep it as simple as possible so I just took what I need it, do you think it's worth it to just import the package for future proofing?